src/uct/api/v2/uct_v2.h

```diff
@@ -187,6 +200,11 @@ typedef struct {
      * This field is set by the UCT layer.
      */
     size_t              max_inflight_eps;
+
+    /**
+     * Flag of supported performance attributes functionalities
```

Comment:
Performance characteristics of the network interface

Reply:
fixed